eeg_flow/tasks/epochs_evoked.py

```diff
@@ -0,0 +1,434 @@
+        derivatives_folder / (fname_stem + "_step9_d3-novel_evoked-ave.fif"),
+    ]
+
+    locks = lock_files(*derivatives, timeout=timeout)
+
+    try:
+        _behav_prep_epoching(participant, group, task, run)
+    finally:
+        for lock in locks:
+            lock.release()
+        del locks
+    return
+
+
+@fill_doc
+def _behav_prep_epoching(
+    participant: str,
+    group: str,
+    task: str,
+    run: int,
+) -> None:
+    """Compute behav report and output metadata.
+    Parameters
+    ----------
+    %(participant)s
+    %(group)s
+    %(task)s
+    %(run)s
+    """
+    # prepare folders
+    _, derivatives_folder_root, _ = load_config()
+    fname_stem = get_fname(participant, group, task, run)
+    derivatives_folder = get_derivative_folder(
+        derivatives_folder_root, participant, group, task, run
+    )
+
+    # load previous steps
+    # # load raw_fit recording
+    raw = read_raw_fif(
+        derivatives_folder / (fname_stem + "_step7_preprocessed_raw.fif"),
+        preload=True,
+    )
+
+    events = find_events(raw, stim_channel="TRIGGER")
+
+    events_id = dict(standard=1, target=2, novel=3, response=64)
+    row_events = ["standard", "target", "novel"]
+
+    metadata, events, event_id = create_metadata(events, events_id, raw, row_events)
+    FNAME_METADATA = derivatives_folder / (fname_stem + "_step9_a-metadata.csv")
+    metadata.to_csv(FNAME_METADATA)
+
+    num_Hits, num_CorrectRejections, num_Misses, num_FalseAlarms = get_SDT_outcomes(metadata)
+
+    hits = metadata[metadata["response_type"] == "Hits"]
+    response_mean = round(hits["response"].mean(), 5)
+    response_std = round(hits["response"].std(), 5)
+
+    plot_RT(hits, fname_stem, derivatives_folder, response_mean, response_std)
+
+    get_indiv_behav(metadata, num_Hits, num_CorrectRejections, num_Misses, num_FalseAlarms, 
+                    fname_stem, derivatives_folder, response_mean, response_std)
+
+
+    # %%
+    epochs = epoching(raw, events, event_id, metadata)
+
+    reject = get_rejection(epochs)
+
+    epochs = clean_epochs_from_rejection(epochs, reject, fname_stem, derivatives_folder)
+
+    
+    # %%
+    FNAME_CLEANED_EPOCHS = derivatives_folder / (
+        fname_stem + "_step9_c1-cleaned-epo.fif"
+    )
+    epochs.save(FNAME_CLEANED_EPOCHS)
+
+    save_evoked(epochs, event_id, fname_stem, derivatives_folder)
+
+    return raw, events, event_id, metadata
+
+def create_metadata(events, events_id, raw, row_events):
+    """
+    metadata for each epoch shall include events from the range: [0.0, 1.5] s,
+    i.e. starting with stimulus onset and expanding beyond the end of the epoch
+    """
+    metadata_tmin, metadata_tmax = 0.0, 0.999
+
+    """
+    auto-create metadata
+    this also returns a new events array and an event_id dictionary.
+    we'll see later why this is important
+    """
+
+    metadata, events, event_id = make_metadata(
```

Comment:
Since MNE calls this kind of function `make_metadata`, let's call it also `make_metadata` and let's rename the MNE one to `make_metadata_mne`, for consistency.
This kind of consistency helps to re-read a code in a couple of months ;)